src/python/pants/backend/go/subsystems/golang.py

```diff
@@ -85,6 +87,8 @@ class GoRoot:
     path: str
     version: str
 
+    raw_metadata: FrozenDict[str, str]
```

Comment:
Maybe make this private with `_`? Probably best for folks to use the properties rather than this.

Reply:
Done.